src/data/multiset/powerset.lean

```diff
@@ -236,4 +236,21 @@ theorem powerset_len_mono (n : ℕ) {s t : multiset α} (h : s ≤ t) :
 le_induction_on h $ λ l₁ l₂ h, by simp [powerset_len_coe]; exact
   ((sublists_len_sublist_of_sublist _ h).map _).subperm
 
+@[simp] theorem powerset_len_empty {α : Type*} (n : ℕ) {s : multiset α} (h : s.card < n) :
+  powerset_len n s = 0 :=
+card_eq_zero.mp (nat.choose_eq_zero_of_lt h ▸ card_powerset_len _ _)
+
+@[simp]
+lemma powerset_len_card_add (s : multiset α) {i : ℕ} (hi : 0 < i) :
+  s.powerset_len (s.card + i) = 0 :=
+powerset_len_empty _ (lt_add_of_pos_right (card s) hi)
+
+@[simp] theorem powerset_len_map {β : Type*} (f : α → β) (n : ℕ) (s : multiset α) :
+  powerset_len n (s.map f) = (powerset_len n s).map (map f) :=
```

Comment:
I'm not sure which side (LHS/RHS) is simpler.

Reply:
I'm not 100% sure either, I only have it this way as this is the way that seemed to be useful in https://github.com/leanprover-community/flt-regular/blob/master/src/number_theory/cyclotomic/absolute_value.lean

Reply:
Ok I removed this simp